lemur/certificates/views.py

```diff
@@ -1074,6 +1078,7 @@ def post(self, certificate_id, data=None):
         log_service.create(g.current_user, "update_cert", certificate=cert)
         return cert
 
+    @operator_permission.require(http_exception=403)
```

Comment:
we have ALLOW_CERT_DELETION config property to disallow deletion across the board, not sure if there's a case where we want to prevent owners from deleting their certs, but we'd want operators to have that ability